src/main/kotlin/org/rust/ide/inspections/fixes/InitializeWithDefaultValueFix.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.inspections.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.NavigatablePsiElement
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.*
+import org.rust.lang.core.psi.ext.RsElement
+import org.rust.lang.core.psi.ext.ancestorOrSelf
+import org.rust.lang.core.resolve.knownItems
+import org.rust.lang.core.types.declaration
+import org.rust.lang.core.types.type
+
+class InitializeWithDefaultValueFix(
+    element: RsElement,
+    private val declaration: RsLetDecl,
+    private val variableName: String,
+    private val typeReference: RsTypeReference
+) : LocalQuickFixAndIntentionActionOnPsiElement(element) {
+    override fun getText() = "Initialize with a default value"
+    override fun getFamilyName() = name
+
+    override fun invoke(project: Project, file: PsiFile, editor: Editor?, startElement: PsiElement, endElement: PsiElement) {
+        val psiFactory = RsPsiFactory(project)
+        val defaultValueFactory = RsDefaultValueBuilder(declaration.knownItems, declaration.containingMod, psiFactory)
```

Comment:
I don't sure that we should use recursive fill here

Reply:
Now `recursive: Boolean = false` by default